bot/requirements-dev.txt

```diff
@@ -4,3 +4,4 @@ pre-commit==1.21.0
 pytest==5.3.2
 pytest-responses==0.4.0
 responses==0.10.9
+tenacity==6.0.0
```

Comment:
It should not be in requirements-dev.txt, but in requirements.txt, since it is used in production code and not only in tests.